pkg/splunk/util/util_test.go

```diff
@@ -179,9 +179,9 @@ func TestPodExecCommand(t *testing.T) {
 
```

Comment:
We have some changes in the testing code to match the new ClusterManager CRD.  In the short, the entire code was switched to Manager CRD, and two new tests were duplicated in `/test/clustermaster` to also validate the ClusterMaster CRD. 

Please let me know if you guys have any comments with regards to the testing code.  Thanks!
@pdhanoyasplunk , @jambrosiano , @tpereirasplunk